### PR TITLE
conddone

### PR DESCRIPTION
**Pull request by shannonfromomaha**

### CODE REVIEW

File: lib/conditionals.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional establishes what should happen when the code incounters a specific input. For example, a conditional could say if name == shannon, then that person is great.
```
Comment (sumeetjain) [inline lib/conditionals.md:8]: Other than checking if `x` is equal to `y`, what are two (or more) kinds of question you can ask in a conditional?

Comment (sumeetjain) [inline lib/conditionals.md:8]: Just making sure you saw the above comment. I can approve this assignment once you address that question.

Comment (shannonfromomaha) [inline lib/conditionals.md:8]: EDIT: Conditionals can ask “is this equal to this?” but they can also ask “is this more than this? less than this? is it capitalized?” and a slew of other questions. (After reading all this information, I realize that I interact with conditionals most obviously when I’m trying to make “safe” passwords for every website on earth). Conditionals are how you make loops, but gives the code an out to make sure that loop is not infinite. It is also the way to make your program seem to have a conversation with you, by making it possible to respond to input appropriately and not just with a set order of answers.

**Comment by sumeetjain** [comment]
:+1:

**Pull request closed without merging**